experimental/adaptive-dialog/docs/all-about-interruption.md

```diff
@@ -67,33 +67,24 @@ Interruption is controlled via the `AllowInterruption` property on input actions
-3. Consider adding an 'interruption' intent that includes examples of things users can say that would be better served by a different dialog within your bot handling that user input. **Do not** add an intent action in that specific dialog to handle this. By not doing this, your dialog is signalling that consultation should move up to the next level up the parent dialog chain. 
-    - In the example listed [here](#global_interrupt), this would cover things like user saying 'How is the weather there next thursday' which would be better served by the `weather` dialog handling it rather than it being considered as a valid response to the bot asking for departure city.
+`AllowInterruptions` defaults to `true`. To turn off interruptions for a specific input, simply set `AllowInterruption = false`.
+
+- To allow interruptions only when a specific intent is detected `AllowInterruptions = #Interruption_intent`
+- To allow interruptions for all intents except a specific intent `AllowInterruptions = !#Input_intent`
+- To allow interruptions only if specific entities are not detected `AllowInterruption = !@entity1 || !@entity2`
+- To allow interruptions only if we have a high confidence intent recogition `AllowInterruption = #Interruption_intent && #Interruption_intent.Score >= 0.8`
 
-**Note:** AllowInteruption = NotRecognized mode does not help in cases where intents you would like to fulfill within the child dialog or up the parent dialog chain expects values of the specific type the active input action is prompting user for.
 
 ## Interruption behavior
-With `AllowInterruption = Always` or `AllowInterruption = NotRecognized`, the active input action will initiate consultation up its parent dialog chain starting with the immediate parent dialog that contains this active input action. 
+When the `AllowInterruptions` property evaluates to `true`, the active input action will initiate consultation up its parent dialog chain starting with the immediate parent dialog that contains this active input action. 
 
 Consultation **stops** when an intent event up the parent dialog chain (including the immediate) parent fires. 
 
-The active input will assume the user utterance to be consumed if there are any actions that are excuted in the consultaton bubble. 
+The active input will assume the user utterance to be consumed if there are any actions that are excuted in the consultaton bubble. When this happens, the active input will always issue a `re-prompt` and not try to re-process the activity for that turn of the conversaiton. To override this, you can set `turn.activityProcessed = false` in the interruption to have the active input re-recognize the activity for that turn of the conversation.
```

Comment:
**Bold** is for suggested updates
"...n the consultaton bubble **path** ..."

..."turn.activityProcessed = false` in the interruption **handler** to have the active  ..."

Reply:
Fixed.